webpack.config.ts

```diff
@@ -29,6 +27,10 @@ const config: Configuration = {
 	},
 	module: {
 		rules: [
+			{
+				test: /readme\.md?$/,
+				loader: './build/readme.loader.cts',
```

Comment:
I'm using .cts here because .ts produces .js, which is interpreted as ESM due to `type:module`, but webpack’s ts-node `require`s it. This instead produces .cjs